dom.bs

```diff
@@ -6764,31 +6778,25 @@ null.
 <p>The <dfn attribute for="Attr"><code>localName</code></dfn> attribute's getter must return the
 <a for=Attr>local name</a>.
 
-<p>The <dfn attribute for="Attr"><code>name</code></dfn> attribute's getter, and
-<dfn attribute for="Attr"><code>nodeName</code></dfn> attribute's getter, must return the
+<p>The <dfn attribute for="Attr"><code>name</code></dfn> attribute's getter must return the
 <a for=Attr>qualified name</a>.
 
-<p>The <dfn attribute for="Attr"><code>value</code></dfn> attribute's getter,
-<dfn attribute for="Attr"><code>nodeValue</code></dfn> attribute's getter, and
-<dfn attribute for="Attr"><code>textContent</code></dfn> attribute's getter, must return the
+<p>The <dfn attribute for="Attr"><code>value</code></dfn> attribute's getter must return the
 <a for=Attr>value</a>.
 
-<p>The {{Attr/value}} attribute's setter, {{Attr/nodeValue}} attribute's setter, and
-{{Attr/textContent}} attribute's setter, must run these steps:
+<p>To <dfn>set an existing attribute value</dfn>, given an <a>attribute</a> <var>attribute</var> and
+string <var>value</var>, run these steps:
 
 <ol>
- <li>If <a>context object</a>'s
- <a for=Attr>element</a> is null, set
- <a>context object</a>'s <a for=Attr>value</a> to the
- given value.
+ <li>If <var>attribute</var>'s <a for=Attr>element</a> is null, then set <var>attribute</var>'s
+ <a for=Attr>value</a> to <var>value</var>.
 
- <li>Otherwise, <a lt="change an attribute">change</a> the
- <a>context object</a> from <a>context object</a>'s
- <a for=Attr>element</a> to the given value.
+ <li>Otherwise, <a lt="change an attribute">change</a> <var>attribute</var> from
```

Comment:
I don't like the call site stating the types. And "an" makes it even worse I think since these are very specific instances. Perhaps at some point we can change the way this is done, or adopt the abstract operation syntax from ECMAScript (I think I'd favor that).